ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -49,4 +53,24 @@ export default class ScopesScopeProjectsRoute extends Route {
     );
     return projects;
   }
+
+  @runEvery(2000)
```

Comment:
Not sure if I should still be using `ember-pollster`, did we ever have any plans to get off of this?

Reply:
I will prefer to get off `ember-pollster`. But TBH I do think that could be done in a separate PR/context since is used in our codebase in other places. For now, I do think is totally fine to keep using it and later on we can consider a refactor?

As long as I know there were no plans for that, maybe something we can ask on tech-frontend how other teams are dealing with this and bring the topic to tech time.